doc/sphinx/introduction.rst

```diff
@@ -120,136 +120,193 @@ contained in the :ref:`Developers guide`.
 Basic python simulation script
 ------------------------------
 
-In this section, a brief overview is given over the most important
-components of the Python interface and their usage is illustrated by
-short examples. The interface is contained in the espressomd Python
-module, which needs to be imported, before anything related can be done.
+In this section, a brief overview is given over the most important components
+of the Python interface. Their usage is illustrated by short examples, which
+can be put together to a demo script. 
 
-::
+.. rubric:: Imports
 
-    import espressomd
-
-Access to the simulation system is provided via the System class. As a
-first step, an instance of the class needs to be created
+As usual, the Python script starts by importing the necessary modules.  The
+|es| interface is contained in the espressomd Python module, which needs to be
+imported, before anything related can be done. ::
 
-::
+    import espressomd
 
-    system=espressomd.System()
+This should be followed by further necessary imports: ::
 
-Note that only one instance of the System class can be created, due to
-limitations in the simulation core. Properties of the System class are
-used to access the parameters concerning the simulation system as a
-whole, , the box geometry and the time step
+    from espressomd.interactions import HarmonicBond
+    from espressomd.electrostatics import P3M 
 
-::
+.. rubric:: espressomd.System
 
-    system.box_l =(10.0,10.0,15.0) print system.time_step
+Access to the simulation system is provided via the System class. As a
+first step, an instance of this class needs to be created. ::
 
-The particles in the simulation are accessed via the ParticleList class.
-It is used to retrieve individual particles of the simulation as well as
-for adding particles. An instance of the class is provided as the part
-attribute of the System class. Individual particles can be retrieved by
-their numerical id by using angular brackets
+    system = espressomd.System(box_l = [10,10,10])
 
-::
+Note that only one instance of the System class can be created due to
+limitations in the simulation core. :ref:`Properties of the System
+class<Setting global variables in Python>` are used to access the parameters
+concerning the simulation system such as box geometry, time step or :ref:`cell-system<Cellsystems>`: ::
 
-    p=system.part[0]
+    print("The box dimensions are {}".format(system.box_l))
+    system.time_step = 0.01
+    system.cellsystem.skin = 0.4
 
-It is also possible to loop over all particles
+.. rubric:: Particles
 
-::
+The particles in the simulation are accessed via the ParticleList class. Use
```

Comment:
"The particles in the simulation are accessed via the object system.part. Use the `add` method of this object to :ref:`create new particles<Adding particles>`: ::"
The object is an instance of the class.